config-model/src/test/java/com/yahoo/searchdefinition/processing/DictionaryTestCase.java

```diff
@@ -0,0 +1,138 @@
+// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+package com.yahoo.searchdefinition.processing;
+
+import com.yahoo.searchdefinition.Search;
+import com.yahoo.searchdefinition.SearchBuilder;
+import com.yahoo.searchdefinition.derived.AttributeFields;
+import com.yahoo.searchdefinition.document.Dictionary;
+import com.yahoo.searchdefinition.parser.ParseException;
+import com.yahoo.vespa.config.search.AttributesConfig;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+/**
+ * Test configuration of dictionary control.
+ *
+ * @author baldersheim
+ */
+public class DictionaryTestCase {
+    private static AttributesConfig getConfig(Search search) {
+        AttributeFields attributes = new AttributeFields(search);
+        AttributesConfig.Builder builder = new AttributesConfig.Builder();
+        attributes.getConfig(builder);
+        return builder.build();
+    }
+    @Test
+    public void testDefaultDictionarySettings() throws ParseException {
+        String def =
+                "search test {\n" +
```

Comment:
Consider using TestUtil.joinLines() to avoid explicit newline characters.

Reply:
Fixed